docs/language/ql-handbook/expressions.rst

```diff
@@ -279,6 +279,21 @@ The following aggregates are available in QL:
   evaluates to the empty set (instead of defaulting to ``0`` or the empty string).
   This is useful if you're only interested in results where the aggregation body is non-trivial.
 
+.. index: unique
+
+- ``unique``: The value of this aggregate depends on how many different values ``<expression>`` can have,
```

Comment:
This reads a bit oddly and doesn't match the wording in the other cases. The best I could come up with is:

```
This aggregate depends on the values of ``<expression>`` over all possible over all possible assignments to the aggregation variables. If there is a unique value of the ``expression`` over the aggregation variables then the aggregate evaluates to that value. Otherwise the aggregate has no value.
```